scripts/jest/jest-setup.js

```diff
@@ -1 +1,5 @@
 import '@testing-library/jest-dom';
+
+global.setImmediate = callback => {
+    setTimeout(callback, 0);
+};
```

Comment:
Had to mock `setImmediate` with a `setTimeout`(0) otherwise the tests just hang there